src/stylesheets/elements/_buttons.scss

```diff
@@ -7,7 +7,7 @@ $button-stroke: inset 0 0 0 2px;
 // This mixin is only needed until we fully remove the
 // deprecated usa-button-disabled style.
 @mixin disabledesque {
-  background-color: $color-gray-lighter;
+  background-color: color('disabled-light');
```

Comment:
I don't even know if disabled buttons are a good UX pattern or not, but I can barely read these. While I can read the existing ones better now. I know that these don't have to pass WCAG.